numba/core/pylowering.py

```diff
@@ -170,6 +167,15 @@ def lower_inst(self, inst):
         else:
             raise NotImplementedError(type(inst), inst)
 
+    @utils.cached_property
+    def _omitted_typobj(self):
+        """Return a `OmittedArg` type instance as a LLVM value suitable for
+        testing at runtime.
+        """
+        from numba.core.dispatcher import OmittedArg
```

Comment:
this here due to circular import problems?

Reply:
yes, confirmed. can't move it to module scope